app/models/course/condition.rb

```diff
@@ -3,4 +3,6 @@ class Course::Condition < ActiveRecord::Base
 
   belongs_to :course, inverse_of: false
   belongs_to :conditional, polymorphic: true
+
+  delegate :satisfied_by?, to: :actable
```

Comment:
Yes, that's abstract correct? Why not just let that abstract implementation delegate to its derived implementation?